### PR TITLE
Add OCI platform setup to documentation

### PR DESCRIPTION
**Pull request by jlbutler**
Add OCI OKE (Container Engine for Kubernetes) to the list of platforms with setup instructions for Istio.

### CODE REVIEW

**Comment by jlbutler** [comment]
Per bot suggestion, assigning to @rcaballeromx

/assign @rcaballeromx

File: content/docs/setup/kubernetes/quick-start/index.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ To install and configure Istio in a Kubernetes cluster, follow these instruction
 1. [Kubernetes platform setup](/docs/setup/kubernetes/platform-setup/):
   * [Minikube](/docs/setup/kubernetes/platform-setup/minikube/)
   * [Google Container Engine (GKE)](/docs/setup/kubernetes/platform-setup/gke/)
+  * [Oracle Cloud Infrastructure (OKE)](/docs/setup/kubernetes/platform-setup/oci/)
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start/index.md]: Would you mind alphabetically sorting this list?

Comment (jlbutler) [reply content/docs/setup/kubernetes/quick-start/index.md]: I don't mind at all. This ends up mixing in non-cloud providers with cloud providers. The list renders on docs/setup/kubernetes/platform-setup in a way that groups the cloud providers together, which I could also do. For now, it's just alphabetized, let me know which you'd prefer.

**Comment by jlbutler** [comment]
Thanks @geeknoid. Is there a way to track when this will be released? And, if that's very far in the future, can I file a PR against the current release to add it in sooner?

**Comment by geeknoid** [comment]
It'll be several more weeks before 1.1 is release, which is when preliminary.istio.io will go live on istio.io.

You can send a PR to the release-1.0 branch to have this published to istio.io immediately.

**Pull request merged**